substrate/frame/vesting/src/lib.rs

```diff
@@ -411,7 +414,7 @@ pub mod pallet {
 		) -> DispatchResult {
 			let who = ensure_signed(origin)?;
 			if schedule1_index == schedule2_index {
-				return Ok(())
+				return Ok(());
```

Comment:
Please undo opinionated formatting (unless required by CI).

Reply:
Done. These were added by `cargo fmt`, but I see `trailing_semicolon` is false in `.rustfmt.toml`, so I'm not sure why that happened.